src/app/modules/main/browser_section/module.nim

```diff
@@ -38,16 +40,19 @@ proc newModule*(delegate: delegate_interface.AccessInterface,
   result.providerModule = provider_module.newModule(result, settingsService, providerService)
   result.bookmarkModule = bookmark_module.newModule(result, bookmarkService)
   result.dappsModule = dapps_module.newModule(result, dappPermissionsService, walletAccountService)
+  result.currentAccountModule = current_account_module.newModule(result, walletAccountService)
```

Comment:
@jrainville it will also register a component for the qml so that won't work as there will be 2 times the same component being registered, ok I could pass a flag, but I really doubt making those name dynamic will help us

Reply:
> @Khushboo-dev-cpp @jrainville I also tried to move the view only in shared part, but it is dependent of delegate and interface are not shared so it becomes a nightmare to just make it compile.
> 
> I will follow this principle here: no abstraction is better than the wrong one! (clearly anything I am doing seems wrong)
> 
> Also as said, let s try to improve that on cpp app

I am okay with that. can we leave a To-do?

Reply:
>  that won't work as there will be 2 times the same component being registered

Ah I forgot about that. Ok then let's leave it as is for now